.prettierrc

```diff
@@ -1,6 +0,0 @@
-{
```

Comment:
Can we bring back this file? And reformat the examples and the code using this Prettier config?

Reply:
Oops, I thought it was lacking TypeScript support. Looks like all we have to do is upgrade prettier.

Reply:
I have restored prettier and integrated it with ESLint to prevent configuration conflict. Note that we might have to disable several ESLint style rules in the future. It is now prettier for style rules + ESLint for the remaining rules.